benchmark/opperf/results/mxnet_operator_benchmark_results_gpu.md

```diff
@@ -0,0 +1,322 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# MXNet GPU Operator Benchmarks
+
+## Settings
+
+1. MXNet - v1.5.0 with CUDA 10.1
```

Comment:
I suppose the version and GPU flags are auto generated in the report. Is that right?

Reply:
Addressed the same. Thanks for pointing at Runtime features. Using it to automate the generation of runtime/compile time flags set for this benchmark runs.